docs: Search Metadata Batch 04 (#14596)
hashicorp/consul: website/content/docs/connect/proxies/index.mdx

---
layout: docs
page_title: Service Mesh Proxy Overview
description: >-

> Consul deploys sidecar proxies to services, allowing them to connect to the service mesh without modifying the underlying application code. You can use the built-in proxy, Envoy, or a custom proxy to handle communication and verify TLS connections.

Reviewer: Consul may or may not deploy the proxies for you (depending on your runtime).
```suggestion
In Consul service mesh, each service has a sidecar proxy that secures connections with other services in the mesh without modifying the underlying application code. You can use the built-in proxy, Envoy, or a custom proxy to handle communication and verify TLS connections.
```
